bin/ui.rb

```diff
@@ -0,0 +1,104 @@
+module UI
+puts "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+puts "X   WELCOME TO TIC TAC TOE!   X"
+puts "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+
+WIN_COMBINATIONS = [[0,1,2], [3,4,5], [6,7,8], 
+                    [0,3,6], [1,4,7], [2,5,8], 
+                    [0,4,8], [2,4,6]]          
+
+def add_player(player)
```

Comment:
Should this method not be part of Player class? Now I wonder why you even have the Player and Gameboard class 🤔